<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>50</y>
      <width>891</width>
      <height>531</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>PLAYERS</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>0</y>
        <width>911</width>
        <height>441</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date of birth</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Position</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nationality</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Club</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Weight</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Height</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>250</y>
        <width>628</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>CLUBS</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>351</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Arsenal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tottenham</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Manchester City</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Leicester City</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Crystal Palace</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Everton</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Burnley</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Southampton</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bournemouth</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Manchester United</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Liverpool</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Chelsea</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>West Ham United</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Watford</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Newcastle United</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cardiff City</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fulham</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Brighton &amp; Hove Albion</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Huddersfield Town</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wolverhampton Wanderers</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="labelClubname">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Club Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelClubcity">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>City:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelManager">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Manager:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineClubname">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineClubcity">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>70</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineManager">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>110</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelClubname_2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>0</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Players List:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>20</y>
        <width>281</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>FIGURES</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>581</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>151</width>
        <height>52</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="text">
          <string>SORT BY MONTH</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentText">
          <string>January</string>
         </property>
         <item>
          <property name="text">
           <string>January</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Febuary</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>189</width>
        <height>52</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="text">
          <string>SORT BY CLUBS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentText">
          <string>Arsenal</string>
         </property>
         <item>
          <property name="text">
           <string>Arsenal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Aston Villa</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AFC Bournemouth</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Brighton and Hove Albion</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Burnley</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chelsea</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Crystal Palace</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Everton</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Leicester City</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Liverpool</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Manchester City</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Manchester United</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Newcastle United</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Norwich City</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sheffield United</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Southampton</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tottenham Hotspur</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Watford</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>West Ham United</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wolverhampton Wanderers</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>TABLES</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>731</width>
        <height>481</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>8</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>9</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>13</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>14</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>15</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>16</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>17</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>18</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>20</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Teams</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Point</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Wins</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Draws</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Loses</string>
       </property>
      </column>
      <item row="0" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>160</width>
      <height>229</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>70</y>
      <width>131</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>ADD PLAYER</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>DELETE PLAYER</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>SEARCH</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>SAVE TO FILE</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1016</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPremier_league">
    <property name="title">
     <string>Premier league</string>
    </property>
   </widget>
   <addaction name="menuPremier_league"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
